source/edge-server/configure.txt

```diff
@@ -113,8 +113,19 @@ server, or hardware that you provision on-site.
 Required Network Access
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-The host that runs the Edge Server must allow access to port 80 via HTTP 
-to the general internet. 
+To accept connections from sync clients or connections via the wireprotocol, 
+the host that runs the Edge Server must allow access to incoming connections.
+
+While Edge Server is in Preview, you can stand up a demo using EC2 or another 
+cloud provider by opening port 80 via HTTP to the general internet. 
+In production, we recommend opening port 443 with SSL.
```

Comment:
There's an implicit suggestion that Preview === dev environment. Do we know if this is true? Beyond that, is using HTTP in a dev environment going away after Edge Server moves out of Preview? If not, we may want to refer to "using Edge Server in a development environment" instead of "While Edge Server is in Preview". Also, maybe we should address the "general internet vs owned infrastructure" a bit earlier?

Here's something to show what I'm getting at:

```suggestion
If you need to open Edge Server to the genera internet, you should open different ports with different methods appropriate for your environment.

- Development. Use port 80 and HTTP.
- Production. Use port 443 and SSL.
```

Reply:
Good call - reworded and reorganized this section a little.